packages/components/src/Alert/Alert.tsx

```diff
@@ -93,20 +94,27 @@ export const Alert = ({
   onActionClick,
   actionText,
   children,
+  customIcon,
   ...props
 }: IAlertProps) => (
   <Container $type={type} {...props}>
     <IconContainer>
-      {type === 'success' && <Check />}
-      {type === 'warning' && <Help />}
-      {type === 'error' && <NotificationError />}
-      {type === 'info' && <Notification />}
-      {type === 'loading' && (
-        <Spinner
-          id="in-progress-floating-notification"
-          baseColor={colors.white}
-          size={20}
-        />
+      {!customIcon ? (
+        <>
+          {type === 'success' && <Check />}
+          {type === 'warning' && <Help />}
+          {type === 'error' && <NotificationError />}
+          {type === 'info' && <Notification />}
+          {type === 'loading' && (
+            <Spinner
+              id="in-progress-floating-notification"
+              baseColor={colors.white}
+              size={20}
+            />
+          )}
+        </>
+      ) : (
+        <>{customIcon}</>
```

Comment:
Nice addition. Please add a note to `packages/components/stories/releases.stories.mdx` under `1.3.0` , something like: "✨ Allow adding custom icon to [Alert](?path=/docs/data-alert--success)"